### PR TITLE
Merge pull request #2276 from pulumi/add-alicloud-docs

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

**Comment by justinvp** [comment]
Welp, I guess we're going to need to fix Pulumify for real...

> Invoking sync function 'arn:aws:lambda:us-east-1:894850187425:function:content-syncer-copyfunc-d52ef89' failed [Unhandled]: "{\"errorMessage\": \"[Errno 28] No space left on device\"

File: content/docs/reference/pkg/_index.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Each cloud vendor has a dedicated package for deploying resources to it:
     * [**Kubernetes Extensions** (`@pulumi/kubernetesx`) <span class="badge badge-preview">PREVIEW</span>]({{< relref "nodejs/pulumi/kubernetesx" >}}) - simpler interface for working with Kubernetes
 
 * [**Aiven** (`@pulumi/aiven`)]({{< relref "nodejs/pulumi/aiven" >}})
+* [**AliCloud** (`@pulumi/alicloud`)]({{< relref "nodejs/pulumi/alicloud" >}})
```
Comment (joeduffy) [inline content/docs/reference/pkg/_index.md]: Should we list this as "Alibaba Cloud"?

File: content/docs/reference/pkg/nodejs/pulumi/alicloud/_index.md [hunk 2]
```diff
@@ -0,0 +1,875 @@
...
+linktitle: "@pulumi/alicloud"
+meta_desc: "Explore members of the @pulumi/alicloud package."
+git_sha: "9546b29284c3141a4c805ab6282e4f27a9e31e90"
+---
+
+<!-- WARNING: this page was generated by a tool. Do not edit it by hand. -->
+<!-- To change it, please see https://github.com/pulumi/docs/tree/master/tools/tscdocgen. -->
+
+
+> This provider is a derived work of the [Terraform Provider](https://github.com/terraform-providers/terraform-provider-alicloud)
```
Comment (joeduffy) [inline content/docs/reference/pkg/nodejs/pulumi/alicloud/_index.md]: Just something to stick in the back of your head, might make sense to try to write a little README, no matter how small, anytime we bring up a new provider. Even if just a link to the docs for configuration, etc.

**Comment by stack72** [comment]
Thanks @joeduffy :)

**Pull request merged**